lib/view_component/slotable_v2.rb

```diff
@@ -148,6 +149,7 @@ def renders_one(slot_name, callable = nil)
       #   <% end %>
       def renders_many(slot_name, callable = nil)
         validate_plural_slot_name(slot_name)
+        validate_singular_slot_name(ActiveSupport::Inflector.singularize(slot_name).to_sym)
 
         singular_name = ActiveSupport::Inflector.singularize(slot_name)
```

Comment:
```suggestion
        singular_name = ActiveSupport::Inflector.singularize(slot_name)
        validate_plural_slot_name(slot_name)
        validate_singular_slot_name(ActiveSupport::Inflector.singularize(slot_name).to_sym)
```
Minor, but could we re-use `singular_name` right below?